src/eos-companion-app-integration-helper.c

```diff
@@ -500,10 +499,6 @@ list_application_infos (GCancellable  *cancellable,
           if (!examine_flatpak_metadata (flatpak_directory, &app_name, &runtime_name, error))
             return NULL;
 
-	  /* Blacklist com.endlessm.encyclopedia.* */
-          if (g_str_has_prefix (app_name, ENCYCLOPEDIA_APP_PREFIX))
-            continue;
-
```

Comment:
Because the code path is executed for the search as well and we do remove Encyclopedia article matches from the search results, but they should appear there.